open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb

```diff
@@ -61,67 +61,78 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    passenger_classes = np.sort(df_train["Pclass"].unique())
</pre><p>line 1 to 5 could be a function?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/173/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
done